Fix WPT logo size differences

mosaadm opened the pull request:
Fix for https://github.com/web-platform-tests/wpt/issues/19326
I changed the 5 svg files to be exactly the same code only different is `fill='color'` which is different depending on color needed and the files with background have one extra line which is a rect that is filled with background color 
```
// second line in files is the one for background color
<rect x="0" y="0" fill="#003C56" width="398" height="340"/>  

```
Files without background were not changed as they are already the same size

mosaadm committed:
Fix WPT logo size differences

foolip commented:
@mosaadm can you say how you made the changes? Do the different variants all have exactly the same size and path data now?

mosaadm commented:
> @mosaadm can you say how you made the changes? Do the different variants all have exactly the same size and path data now?

@foolip I updated PR description, basically i just copy-pasted code in`wpt-logo-darkblue.svg` and changed the colors
